tests/cross_functional/system_test/test_graceful_nodes_shutdown.py

```
@@ -460,35 +465,49 @@ def test_graceful_nodes_shutdown(
         time.sleep(600)
 
-        # check cluster health
+        logger.test_step("Validate cluster health after node restart")
         try:
-            logger.info("Making sure ceph health is OK")
+            logger.info("Checking Ceph health status")
             Sanity().health_check(tries=50, cluster_check=False)
+            logger.info("Cluster health check passed")
         except Exception as ex:
-            logger.error("Failed at cluster health check!!")
+            logger.error("Cluster health check failed")
```

Ruff (0.15.21)

[warning] Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)